---
layout: post
title: "Genotype–phenotype associations with single-cell data"
author: Dr. Yongjian Yang, Texas A&M， USA
date: 2023-06-01
categories: news
image: images/Yongjian_Yang.png
tags: AI, Ensemble,Deep Learning, image, synthetic data, R&D, pharmaceutical, biomedicine, Long-tailed learning, Deep Learning,single-cell
---

- Title：MAIB-Talk-018: Computational learning genotype–phenotype associations with single-cell data
- Date：10:00pm US East time, 06/24/2023
- Date：10:00am Beijing time, 06/25/2023
- Zoom  ID：933 1613 9423
- Zoom PWD：416262
- Zoom: [https://uwmadison.zoom.us/meeting/register/tJcudu-prTIuGNda1MsF8PKyRQlnGn06TP2E](https://uwmadison.zoom.us/meeting/register/tJcudu-prTIuGNda1MsF8PKyRQlnGn06TP2E)

Presentation Record(Previous Presentation will be showed here if the video is not released for this talk)

<p align="center">
<iframe width="560" height="315" src="https://www.youtube.com/embed/YC1GhRFNs8U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</p>

Dr. Yongjian Yang

Yongjian is a fourth-year doctoral student in the Department of Electronics and Computer Science, Texas A&M University. The supervisor is Dr James Cai. His research direction is computational biology and bioinformatics. Phenotypic associations provide biological explanations, including but not limited to cell communication, gene knockouts, and gene-protein associations. His related work has been published in journals such as Cell Systems, Nucleic Acids Research and academic conferences such as RECOMB and ICIBM.

Abstract

We are developing and applying machine learning models on biomedical data, especially single-cell data, to provide biological interpretation for genotype-phenotype associations. In the first half of my presentation, I would introduce scTenifoldXct, a semi-supervised computational tool for detecting ligand-receptor (LR)-mediated cell-cell interactions and mapping cellular communication graphs. Our method is based on manifold alignment, using LR pairs as inter-data correspondences to embed ligand and receptor genes expressed in interacting cells into a unified latent space. Neural networks are employed to solve the optimization problem. In the second half, I would like to introduce GenKI, a virtual knockout (KO) tool for gene function prediction using scRNA-seq data in the absence of KO samples when only wild-type (WT) samples are available. Without using any information from real KO samples, GenKI, which adapts a variational graph autoencoder (VGAE) model, is designed to capture shifting patterns in gene regulation caused by the KO perturbation in an unsupervised manner and provide a robust and scalable framework for gene function studies. 

Reference

Yang, Yongjian, et al. "scTenifoldXct: A semi-supervised method for predicting cell-cell interactions and mapping cellular communication graphs." Cell Systems 14.4 (2023): 302-311.

Yang, Yongjian, et al. "Gene knockout inference with variational graph autoencoder learning single-cell gene regulatory networks." Nucleic Acids Research (2023): gkad450.