---
layout: post
title: "MAIB-talk-011: SCALE: Single-cell ATAC-seq analysis via latent feature extraction"
author: Dr. Lei Xiong
date: 2023-03-16
categories: news
image: images/Lei_Xiong_MIT.png
tags: Single-cell; ATAC-seq; Epigenetic; Clustering; Imputation
---

- Speaker： Dr. Lei Xiong (熊磊) @ MIT, Massachusetts Institute of Technology
- Title： MAIB讲座第11期: SCALE: Single-cell ATAC-seq analysis via latent feature extraction
- Date：10:00pm US East time, 03/24/2023
- Date：10:00am Beijing time, 03/25/2023
- Zoom  ID：933 1613 9423
- Zoom PWD：416262
- URL: https://uwmadison.zoom.us/meeting/register/tJcudu-prTIuGNda1MsF8PKyRQlnGn06TP2E

Title: 

SCALE: Single-cell ATAC-seq analysis via latent feature extraction

Bio: 

熊磊，MIT 博士后，博士毕业于清华大学生命学院。主要研究方向是利用深度学习技术开发单细胞基因组分析工具。主要工作发表在Nature Communications期刊上。

Background:

细胞异质性是生命的普遍特征，广泛影响着包括发育、神经和肿瘤等生命现 象。对于多细胞个体而言，尽管所有细胞的基因组是相同的，但是由于调控基因 表达的表观基因组不同，使得不同细胞的基因差异表达，从而呈现出不同形态功 能。因此，从表观基因组层面理解并揭示细胞异质性的机制非常重要。
单细胞技术可以准确高效地分离出单个细胞，并结合高通量测序技术，可以 检测单细胞精度的 DNA、RNA 和蛋白质等层面的细胞差异，广泛应用于细胞异质 性的研究。近年来不断发展的单细胞 ATAC-seq(scATAC-seq)技术借助 Tn5 转座 酶在单细胞水平上特异性地标记并检测染色质的开放区间，揭示了基因表观调控 层面的细胞间的差异。
然而，由于高通量技术的限制，无法有效检测每一个单细胞的所有信号;此 外，二倍体基因组的染色质上开放位点一般最多只有两个拷贝，而整个基因组的 能检测到开放位点的数目有几十万之多，导致许多信号丢失。因此，单细胞 ATAC-seq 数据具有异常的高维度和稀疏性等特点，使得数据分析变得极具挑战。
在本论文中，我们结合了变分自编码器和高斯混合概率模型，开发了对单细 胞ATAC-seq数据进行分析的方法，叫做Single-cell ATAC-seq analysis via Latent feature Extraction(SCALE)。首先，SCALE准确对单细胞ATAC-seq数据的概率 分布进行建模，并通过提取隐层特征的方式，将高维的数据降维至低维空间，解 决了维度灾难问题。由于相同类型的细胞数据类似，我们引入高斯混合模型对细 胞聚类并通过共享参数，学习同类型细胞的共有特征，实现了同类型细胞之间互 相填补缺失信息，从而解决了数据稀疏性问题。
我们在不同实验方法、实验平台来源的不同质量的数据集上验证了 SCALE 在 包括可视化、聚类、填补缺失值等方面的性能都显著地优于其他方法。此外，我 们还发现 SCALE 提取的特征不但可以用来解释细胞类群的生物学功能，还可以用 来揭示单细胞 ATAC-seq 实验中的潜在的批次效应。网址：https://github.com/jsxlei/SCALE

Introduction:

Cellular heterogeneity is a universal characteristic of life that extensively affects various biological processes, including development, neural activity, and tumorigenesis. For multicellular organisms, even though all cells have the same genome, the differences in epigenetic modifications regulating gene expression result in differential gene expression among different cell types, which leads to their distinct morphological and functional features. Therefore, it is crucial to understand and uncover the mechanisms underlying cellular heterogeneity at the epigenomic level.

Single-cell technology can accurately and efficiently isolate individual cells, and when combined with high-throughput sequencing technologies, can detect cell-to-cell differences at the DNA, RNA, and protein levels. It has been widely used in studying cellular heterogeneity. In recent years, the single-cell ATAC-seq (scATAC-seq) technique, which utilizes the Tn5 transposase to specifically label and detect open chromatin regions at the single-cell level, has revealed cell-to-cell differences at the level of gene expression regulation.

However, due to the limitations of high-throughput technologies, it is not feasible to effectively detect all signals in each individual cell. In addition, open chromatin regions in diploid genomes generally have only two copies at most, and the number of detectable open chromatin regions throughout the genome can reach hundreds of thousands, leading to the loss of many signals. As a result, scATAC-seq data has high dimensionality and sparsity, making data analysis extremely challenging.

In this paper, we developed a method called Single-cell ATAC-seq analysis via Latent feature Extraction (SCALE) by combining variational autoencoder and Gaussian mixture probabilistic model to analyze scATAC-seq data. Firstly, SCALE accurately models the probability distribution of scATAC-seq data and reduces the high-dimensional data to a low-dimensional space by extracting latent features, thus solving the problem of the curse of dimensionality. Since data from the same cell type are similar, we introduce a Gaussian mixture model to cluster cells and learn common features among cells of the same type through shared parameters, filling in missing information among cells of the same type and solving the problem of data sparsity.

We validated the performance of SCALE on datasets of varying quality from different experimental methods and platforms, and demonstrated that it outperformed other methods in visualization, clustering, and imputation of missing values. In addition, we found that the features extracted by SCALE not only explain the biological functions of cell populations, but also reveal potential batch effects in scATAC-seq experiments. The code is available at https://github.com/jsxlei/SCALE.


<p align="center">
<iframe width="560" height="315" src="https://www.youtube.com/embed/7NWBLuok8nk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</p>
